File: Sum.java
/*
$ time java Sum
125000000250000000

real	0m0.350s
user	0m0.311s
sys	0m0.031s
*/
import java.*;

public class Sum
{
  public static void main(String[] args)
  { 
    long sum=0;
    int limit=500000001;
    for (int i=0; i<limit; i++)
    { sum+=i;
    }
    System.out.println(sum);
  }
}

File: sum.js
/*
$ time node sum.js
125000000067108900

real	0m0.718s
user	0m0.689s
sys	0m0.022s
*/
var limit = 1+500e6;
var sum = 0;
for (var i=0; i<limit; i++)
{ sum+=i;
}
console.log(sum);

